pkg/fanal/secret/builtin-allow-rules.go

```diff
@@ -4,7 +4,7 @@ var builtinAllowRules = []AllowRule{
 	{
 		ID:          "tests",
 		Description: "Avoid test files and paths",
-		Path:        MustCompile(`(\/test|-test|_test|\.test)`),
+		Path:        MustCompile(`(^test|\/test|-test|_test|\.test)`),
```

Comment:
@knqyf263 
there is a moment with this change.
the tests are failing, because Trivy is skipping `testdata` folder now.

as a workaround we can change `testdata` to `tstdata`, is it ok?

Reply:
oh, this update can break a lot of places...

Reply:
>as a workaround we can change testdata to tstdata, is it ok?

No, `testdata` is defined in Go as ignored dirs.

>The go tool will ignore a directory named "testdata", making it available to hold ancillary data needed by the tests.

https://pkg.go.dev/cmd/go/internal/test

Reply:
ok, i understood it. 
will try to skip this build-in rule